### DISCUSSION TITLE
Different gui views under import-export

### DISCUSSION THREAD

**Discussion by pcfriek1987** [apache/cloudstack#9159] [Q&A]
As following https://github.com/apache/cloudstack/issues/8727 I would like to start this discussion here too.

For some reason I'm having a different view then 2 other people in that issue under the same version.

This is my view:
<img width="1398" alt="CS1" src="https://github.com/apache/cloudstack/assets/10122551/8e48515d-5c3c-409a-b012-e71bb9437012">

This was Vishesh view:
![CS2](https://github.com/apache/cloudstack/assets/10122551/c3c2961c-f076-4a40-a44a-0563f84dcdbc)

The only difference I can see is that I used the cloudstack repo instead of the source, but as it is the same version I would have assumed it to be the same layout and such.

It's not a caching issue as was already determined in the above issue.

**Comment by DaanHoogland** [comment]
@pcfriek1987 , is this still an issue?

**Comment by pcfriek1987** [comment]
I have no idea actually, as I haven't had a need to migrate any vm's and i don't have a server to check this quickly, so for now we can close this up, if needed will create a new issue if needed.
